classes/helper.php

```diff
@@ -61,5 +66,17 @@ public static function enable_csp_header() {
             @header('Content-Security-Policy: ' . $cspheaderenforcing);
         }
     }
-}
 
+    public static function enable_feature_policy() {
+        $settings = get_config('local_csp');
+        if (self::$featurebootstrapped or empty($settings->feature_policy_enable)) {
+            return;
+        }
+        self::$featurebootstrapped = true;
+
+        $featureheader = trim(str_replace(array("\r\n", "\r"), " ", $settings->feature_policy));
```

Comment:
If it is one entry per line I think we need to lock down the delimiter

ie  a policy like this:

```
Feature-Policy: microphone 'none'; geolocation 'none'
```

would have a config value of
```
microphone 'none'
geolocation 'none'
```

so we should add in the `;`

either way should be explicitly in the setting help text

Reply:
Hmm perhaps I worded that badly. The semicolon is always required, but it makes no difference whether they are chained on one line, or seperated by a line break